### PR TITLE
Update community-plugin-marketplace.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor formatting, fixed double spacing between headers.

WIP: Updating the process to include steps around documentation, links to UI guidelines, design process.

### CODE REVIEW

File: site/content/extend/plugins/community-plugin-marketplace.md [hunk 1]
```diff
@@ -4,35 +4,38 @@ subsection: Plugins (Beta)
 weight: 60
 ---
 
-Once your plugin has reached a certain level of quality, you might consider submitting it to the Plugin Marketplace. The Plugin Marketplace is a platform that supports discovery, installation and updates of plugins directly within Mattermost. It's a great way to get feedback on your plugin and help make it more popular. Once your plugin is accepted to the Marketplace, Mattermost will also send you swag!
+# Community Plugins in the Marketplace
```
Comment (hanzei) [inline site/content/extend/plugins/community-plugin-marketplace.md:7]: Is that really needed? The hugo theme should add it by default.

**Review by hanzei — APPROVED** [review]
One question, but the looks good :+1: 

Could you please also update https://github.com/mattermost/mattermost-marketplace/blob/master/.github/ISSUE_TEMPLATE/add_plugin.md after this PR is merged?

**Review by aaronrothschild — APPROVED** [review]
Sorry had this show up blocked from a filter I use for GH issues.  Apoogies.  

LGTM

File: site/content/extend/plugins/community-plugin-marketplace.md (hunk 1 above)
Comment (justinegeffen) [reply site/content/extend/plugins/community-plugin-marketplace.md:7]: I didn't actually know that - thanks!

**Pull request merged**